src/components/Blog/BlogArticlesBoard/BlogArticlesBoard.tsx

```diff
@@ -37,76 +42,171 @@ export const BlogArticlesBoard = ({
+
+    if (openDropdown) {
+      !classList?.contains(ariaLabel) && setOpenDropdown(false);
+    }
+  };
 
   return (
-    <BlogArticlesBoardContainer id="see-all">
+    <BlogArticlesBoardContainer
+      id="see-all"
+      onClick={(event: React.MouseEvent<HTMLDivElement>) =>
+        handleClick({ target: event.target as HTMLElement })
+      }
+    >
       <BlogArticlesBoardTitle variant="headerMedium">
         {t('blog.categories')}
       </BlogArticlesBoardTitle>
       {filteredTags ? (
         <BlogArticlesBoardTabs
+          ariaLabel={ariaLabel}
           openDropdown={openDropdown}
           filteredTags={filteredTags}
-          categoryId={categoryId}
+          tabId={tabId}
         />
       ) : (
         <Skeleton sx={{ width: '100%', height: 68 }} />
       )}
-      <Fade in={!!data} timeout={600}>
-        <></>
-        {/* todo: enable */}
-        {/* <ArticlesGrid container>
-          {!!data ? (
-            Array.from({ length: pageSize }).map((_, index) => (
-              <BlogArticleCardSkeleton
-                key={`blog-article-card-skeleton-${categoryId}-${index}`}
-              />
-            ))
-          ) : data?.length > 0 ? (
-            data?.map((article: BlogArticleData, index: number) => (
+      {/* <Fade in={!!data} timeout={600}> */}
+      {!data ? (
+        Array.from({ length: tags.meta.pagination.pageSize }).map(
+          (_, index) => (
+            <BlogArticleCardSkeleton
+              key={`blog-article-card-skeleton-${tabId}-${index}`}
+            />
+          ),
+        )
+      ) : tags.data?.length > 0 ? (
+        tags.data?.map((tag, tagIndex: number) => {
+          const pagination = {
+            page: 0,
+            pageSize: pageSize,
+            pageCount: Math.ceil(
+              tag.attributes.blog_articles.data.length / pageSize,
+            ),
+            total: tag.attributes.blog_articles.data.length,
+          };
+          return (
+            <CategoryTabPanel
+              value={tabId || 0}
+              index={tagIndex}
+              tags={tags}
+              pagination={pagination}
+              ariaLabel={ariaLabel}
+              data={tag.attributes.blog_articles.data}
+            />
+          );
+        })
+      ) : (
+        <p>No Content</p> //todo: find better option
+      )}
+      {/* </Fade> */}
+    </BlogArticlesBoardContainer>
+  );
+};
+
+interface CategoryTabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+  tags: GetTagsResponse;
+  pagination: StrapiMetaPagination;
+  ariaLabel: string;
+  data: BlogArticleData[];
+}
+
+function CategoryTabPanel({
+  children,
+  value,
+  index,
+  tags,
+  data,
+  pagination,
+}: CategoryTabPanelProps) {
+  const theme = useTheme();
+  const [pageTab, setPageTab] = useState(pagination.page);
+  const chunkedPages = chunkArray(data, pagination.pageSize);
+  return (
+    <Box
+      role="tabpanel"
+      hidden={value !== index}
```

Comment:
Same comment as before

Reply:
The board component is not being used. But I added that change

Reply:
Better to remove it then @dennyscode

Reply:
Having a hard time doing that and got Thomas to agree, that we can keep it.
The Board to allow navigating between categories seems to be a viable option to try at some point. Since we may consider coming back to it, should I still remove it and if needed, find it in the history of Git?

Reply:
Actually, it´s removed now!